### ISSUE TITLE
use %(ENV_HOME)s in supervisord

### ISSUE THREAD

**Issue by luto** [Uberspace/lab#263]
```ini
[program:up1]
directory=%(ENV_HOME)s/up1/server/
command=node %(ENV_HOME)s/up1/server/server.js
```

works just fine and requires less manual work. Update all guides.

**Comment by nichtmax** [comment]
fixed in #280

**Comment by unite42** [comment]
> But why is the an "s" after the variable?

Because of the syntax how enviroment variables are called within supervisord:
[http://supervisord.org/configuration.html?#environment-variables](http://supervisord.org/configuration.html?#environment-variables)
